src/client/components/input/select2.js

```diff
@@ -31,10 +32,21 @@ if (typeof window !== 'undefined') {
 }
 
 class Select extends React.Component {
+
```

Comment:
#nit: Extraneous whitespace. Generally, I'd start the block off with a line containing something. Not hugely important, though.